standalone-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsCli.java

```diff
@@ -0,0 +1,317 @@
+    }
+
+    // Examples region
+
+    private void doSimpleExample() throws IOException {
+        final String prefix = "simple_example > ";
+
+        output("You have selected a simple example to see how Pravega StreamCuts work. Great choice!!%n");
+        output("Now, we will ask you some questions to set up the example: %n%n");
+
+        output("How many streams do you want to create? (e.g., from 1 to %s):%n", exampleMaxStreams);
+        final int numStreams = askForIntInput(prefix, 0, exampleMaxStreams);
+
+        output("How many events per stream do you want to create? (e.g., from 1 to %s):%n", exampleMaxEvents);
+        final int numEvents = askForIntInput(prefix, 0, exampleMaxEvents);
+
+        // Set up and write data in streams.
+        StreamCutsExample example = new StreamCutsExample(numStreams, numEvents, scope, controllerURI);
+        example.createAndPopulateStreamsWithNumbers();
+        System.out.println(example.printStreams());
+
+        output("Your Streams are ready :)%n%n");
+        output("Now, to see how StreamCuts work, we are going to build them!%n");
+
+        // After setting up the streams and populating them, we can exercise StreamCuts multiple times.
+        do {
+            doBoundedPrinting(prefix, numStreams, numEvents, example);
+            output("Do you want to repeat? (Y)%n");
+        } while(readLine("%s", prefix).trim().equalsIgnoreCase("Y"));
+
+        output("Sealing and deleting streams.%n");
+        example.deleteStreams();
+        example.close();
+    }
+
+    private void doBoundedPrinting(String prefix, int numStreams, int exampleNumEvents, StreamCutsExample example) throws IOException {
+        final char streamId = StreamCutsExample.streamBaseId;
+        Map<Stream, StreamCut> startStreamCuts = new LinkedHashMap<>();
+        Map<Stream, StreamCut> endStreamCuts = new LinkedHashMap<>();
+
+        for (char id = streamId; id < streamId + numStreams; id++) {
+            final String streamName = String.valueOf(id);
+            output("[Stream %s]  StreamCut start event number.%n", streamName);
+            int iniEventIndex = askForIntInput(prefix, 0, exampleNumEvents);
+            output("[Stream %s] StreamCut end event number.%n", streamName);
+            int endEventIndex = askForIntInput(prefix, iniEventIndex+1, exampleNumEvents);
+            final List<StreamCut> myStreamCuts = example.createStreamCutsByIndexFor(streamName, iniEventIndex, endEventIndex);
+            startStreamCuts.put(Stream.of(scope, streamName), myStreamCuts.get(0));
+            endStreamCuts.put(Stream.of(scope, streamName), myStreamCuts.get(1));
+        }
+
+        // Next, we demonstrate the capabilities of StreamCuts by enabling readers to perform bounded reads.
+        ReaderGroupConfig.ReaderGroupConfigBuilder configBuilder = ReaderGroupConfig.builder();
+        for (Stream myStream: startStreamCuts.keySet()) {
+            configBuilder = configBuilder.stream(myStream);
```

Comment:
Since we already know the number of streams  and `createStreamCutsByIndexFor(..)` returns StreamCut.UNBOUNDED we could populate the start and end streamcut directly using 
configBuilder.stream(myStream, startStreamCut, endStreamCut).

Reply:
Right. I deleted this code block as the info about Streams is already present in `startStreamCuts` and `endStreamCuts`.